test/Icons.Test/Services/IconFetchingServiceTests.cs

```diff
@@ -14,7 +14,7 @@ public class IconFetchingServiceTests
         [InlineData("neverssl.com")] // http site
         [InlineData("ameritrade.com")]
         [InlineData("icloud.com")]
-        [InlineData("bofa.com")]
+        [InlineData("bofa.com", Skip = "Broken in .NET 6 ???")]
```

Comment:
Remember this test is a little flaky. Bofa has gotten us before when they just decided to stop forwarding.

Reply:
It seems much less flaky and just broken in .NET 6. But I didn't notice much difference in my vault.

Reply:
It seems to only be flaky in the pipeline but works fine locally. I have added a tracking link so that we can address it in the future but it's not a worry for me about upgrading to .NET 6.